storage/db-tool/src/backup_maintenance.rs

```diff
@@ -28,11 +29,14 @@ pub struct CompactionOpt {
     #[clap(long, default_value = "1")]
     pub transaction_file_compact_factor: usize,
     #[clap(flatten)]
-    pub metadata_cache_opt: MetadataCacheOpt,
+    pub metadata_cache_path: MetadataCacheOpt,
     #[clap(flatten)]
     pub storage: DBToolStorageOpt,
     #[clap(flatten)]
-    concurrent_downloads: ConcurrentDownloadsOpt,
+    pub concurrent_downloads: ConcurrentDownloadsOpt,
+    /// Specify how many seconds to keep compacted metadata file before moving them to backup folder
+    #[clap(long, default_value = "86400")]
+    pub expiration_duration: u64,
```

Comment:
help = "Remove metadata files replaced by compaction after specified seconds. They were not replaced right away after compation in case they are being read then."

maybe rename: `--remove-compacted-files-after`

best if we put the other comment lines to `help = ` as well, but I feel this one particularly needs explanation.